.idea/codeStyles/Project.xml

```diff
@@ -1,209 +1,8 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <option name="USE_SAME_INDENTS" value="true" />
```

Comment:
you will have to revert all these changes to these files under `.idea` folder. These are project configuration files to ensure everybody shares the same copyright, codestyle, inspection, etc